code/ui/Hud.cs

```diff
@@ -86,16 +88,51 @@ public GeneralHud()
             }
         }
 
-        public class AliveHud : Panel
+        public class AliveHud
         {
-            public AliveHud()
+            public bool Enabled
```

Comment:
Could technically inherit from panel and then override the `Enabled` behaviour (I'm pretty sure)

Reply:
We don't need multiple panel instances, that would be incompatible anyways as there gonna be issues with z-index and ordering. With this change, you can mix panels with general hud panels, theoretically.

Reply:
yeah nvm, there isn't any reason for this.